docs/ocp_prereqs_powervs.md

```diff
@@ -192,13 +192,14 @@ This usually takes a day to get enabled.
 ## RHCOS and RHEL/CentOS 8.X Images for OpenShift
 RHEL image is used for bastion and RHCOS is used for the OpenShift cluster nodes.
 
-You'll need to create [OVA](https://en.wikipedia.org/wiki/Open_Virtualization_Format) formatted images for RHEL and RHCOS, upload them to IBM Cloud Object storage and then import these images as boot images in your PowerVS service instance.
+You'll need [OVA](https://en.wikipedia.org/wiki/Open_Virtualization_Format) formatted images for RHEL and RHCOS, upload them to IBM Cloud Object storage and then import these images as boot images in your PowerVS service instance.
 
 Further, the image disk should be minimum of 120 GB in size.
 
-### Creating OVA images
+### OVA images
 
 - If you have PowerVC then you can follow the instructions provided in the [link](https://www.ibm.com/support/knowledgecenter/en/SSXK2N_1.4.4/com.ibm.powervc.standard.help.doc/powervc_export_image_hmc.html) to export an existing PowerVC image to OVA image.
+- For 4.11 onwards, you can use the OVA formated images that are available from the following [link](https://releases-rhcos-art.apps.ocp-virt.prod.psi.redhat.com/?stream=releases)
```

Comment:
Sorry, I am not sure why this is needed. This link will need RH VPN to download the files, not every user will have access to it.
Please create an issue describing what and why needs to change.

Reply:
The OVA images are published in public bucket, and can be listed with `ibmcloud cos objects --bucket rhcos-powervs-images-${VPC_REGION} --region ${VPC_REGION} --json`, and import with `ibmcloud pi image-import ${image_name} --bucket-access public --disk-type ${storage_type} --image-file-name ${image_file} --bucket rhcos-powervs-images-${VPC_REGION} --region ${VPC_REGION}  --job`. No need to create OVA from `qcow2` file.